viper-runner/scripts/before_plugin.vpr

```diff
@@ -0,0 +1,7 @@
+field left: Int
+field right: Int
+
+// Before plugin
```

Comment:
Test `.vpr` file for benchmarking what life would be like to exist without the inline predicate plug in.